lib/external_service_new/createsend_dispatcher.rb

```diff
@@ -0,0 +1,41 @@
+require "oj"
+require "rest_client"
+require "external_service_new/rest_client_dispatcher"
+require "createsend"
+
+module ExternalServiceNew
+  class CreatesendDispatcher < RestClientDispatcher
+    #maximum and minimun email fetch by page
+    PAGE_SIZE_MAX = 1000
+    PAGE_SIZE_MIN = 10
+
+    def initialize(account_identifiers: nil, credential_details:, sleep_time_seconds: 5, client: RestClient)
+      account_identifiers = account_identifiers.symbolize_keys
+      credential_details = credential_details.symbolize_keys
+      @auth = {
```

Comment:
another @auth here, have you seen this in other examples? Not sure i like it either. Obligates you to do that hashy stuff and still it makes the #initialize parameters non verbose